crates/modelardb_client/src/main.rs

```diff
@@ -92,29 +94,40 @@ fn main() -> Result<(), String> {
     .map_err(|error| format!("Cannot execute queries: {error}"))
 }
 
-/// Parse the command line arguments in `args` and return a pair with the address of the server to
-/// connect to and the file containing the queries to execute on the server. If one of the command
-/// line arguments is not provided it is replaced with [`None`] in the returned pair.
-fn parse_command_line_arguments(mut args: Args) -> (Option<String>, Option<String>) {
+/// Parse the command line arguments in `args` and return a triple with the host of the server to
+/// connect to, the port to connect to, and the file containing the queries to execute on the
+/// server. If one of these command line arguments is not provided it is replaced with [`None`]."
```

Comment:
I think there is an extra `"` at the end of this line.

Reply:
Fixed.